app/main.py

```diff
@@ -1,3 +1,31 @@
-def cache(func):
-    # Write your code here
-    pass
+import functools
+from typing import Callable, Any, Tuple
+
+
+def cache(func: Callable[..., Any]) -> Callable[..., Any]:
+    cache_data = {}
+
+    @functools.wraps(func)
+    def wrapper(*args: Tuple[Any], **kwargs: Any) -> Any:
```

Comment:
*args is Tuple
but elements can by Any
```suggestion
    def wrapper(*args: Any, **kwargs: Any) -> Any:
```